Formula/ddcctl.rb

```diff
@@ -0,0 +1,16 @@
+class Ddcctl < Formula
+  desc "DDC monitor controls (brightness) for Mac OSX command-line"
+  homepage "https://github.com/kfix/ddcctl"
+  url "https://github.com/kfix/ddcctl/archive/refs/tags/v0.tar.gz"
+  sha256 "8440f494b3c354d356213698dd113003245acdf667ed3902b0d173070a1a9d1f"
+  license :public_domain
+
+  def install
+    system "make"
+    bin.install "#{buildpath}/bin/release/ddcctl"
```

Comment:
```suggestion
    bin.mkpath
    system "make", "install", "INSTALL_DIR=#{bin}"
```
It's almost always better to rely on `make` (or similar tool) to move binary objects to their install directory. It never hurts, and can often help.